File: get_broken_src.js
const { chromium } = require('playwright');
(async () => {
  const browser = await chromium.launch();
  const page = await browser.newPage();
  await page.goto('https://ytuong.fedu.vn');
  
  await page.evaluate(() => {
    let st = JSON.parse(localStorage.getItem('fedu_video_ideas_state') || '{}');
    st.curationMode = true;
    localStorage.setItem('fedu_video_ideas_state', JSON.stringify(st));
  });
  await page.reload();
  await page.waitForTimeout(3000);
  await page.evaluate(() => window.scrollTo(0, document.body.scrollHeight));
  await page.waitForTimeout(3000);
  
  const srcs = await page.evaluate(() => {
    return Array.from(document.querySelectorAll('img')).map(img => img.src);
  });
  
  console.log("All image srcs (first 10):");
  console.log(srcs.slice(0, 10));
  
  await browser.close();
})();

File: get_broken_src.py
import asyncio
from playwright.async_api import async_playwright

async def main():
    async with async_playwright() as p:
        browser = await p.chromium.launch(headless=True)
        page = await browser.new_page(viewport={"width": 1280, "height": 800})
        
        await page.goto("https://ytuong.fedu.vn")
        await page.wait_for_timeout(3000)
        
        srcs = await page.evaluate("""
            Array.from(document.querySelectorAll('img')).map(img => img.src)
        """)
        
        print("All image srcs (first 20):")
        for src in srcs[:20]:
            print(src)
            
        await browser.close()

asyncio.run(main())
